front/dangerfile.ts

```diff
@@ -15,12 +18,41 @@ function warnMigrationAck(migrationAckLabel: string) {
   warn(
     "Files in `**/lib/models/` have been modified and the PR has the `" +
       migrationAckLabel +
-      "` label. Don't forget to run the migration from the `-edge` infrastructure."
+      "` label. Don't forget to run the migration from prodbox."
+  );
+}
+
+function checkDocumentationLabel() {
+  const hasDocumentationAckLabel = danger.github.issue.labels.some(
+    (label) => label.name === documentationAckLabel
+  );
```

Comment:
Let use a generic helper to check if an issue/PR has a label.

Reply:
`nit` Let's try to keep the organise. All the migration logic grouped and same for the documentation.